source/stylesheets/layout/_header.scss

```diff
@@ -0,0 +1,162 @@
+@import "mediaqueries/media-header";
+@import "components/_button";
+
+// import fontello icons
+@import "icons/fontello";
+@import "icons/fontello-ie7";
+@import "icons/fontello-ie7-codes";
+@import "icons/fontello-codes";
+@import "icons/fontello-embedded";
+
+//services navigation styling
+.services-nav {
+  display: flex;
+  justify-content: space-between;
+  margin: $base-margin 0;
+  p {
+    color: $grey;
+    font-weight: 400;
+    margin: $base-margin;
+  }
```

Comment:
Install and configure https://github.com/brigade/scss-lint for style check. Check all files